vhdbuilder/packer/windows/windows_settings.json

```diff
@@ -53,6 +53,20 @@
       "windows_image_name": "windows-23H2",
       "base_image_version": "25398.1425.250210",
       "patches_to_apply": []
+    },
+    "2025": {
+      "os_disk_size": "132",
```

Comment:
Do PMs approve this VHD size for WS2025?

Reply:
No - I had to set this because the minimum for a VM from the marketplace was 127GB. I expect the windows team will drop this limit for us. Currently we're still testing.

Reply:
As per your comments, I've updated this to use the smalldisk version of windows2025.